src/__screenshot_tests__/circle-screenshot-test.tsx

```diff
@@ -12,3 +12,19 @@ test.each(CONTENT_OPTIONS)('Circle - %s', async (content) => {
 
     expect(await story.screenshot()).toMatchImageSnapshot();
 });
+
+test('Circle with custom background', async () => {
+    await openStoryPage({
+        id: 'components-primitives-circle--default',
+        skin: 'O2-new',
+        args: {
+            content: 'none',
+            border: true,
+            size: 100,
+            customBackground: true,
```

Comment:
I'd prefer to allow setting the background style in the arg but storybook makes this impossible because it removes values with characters like "(". So you can't set a background style with an image